### ISSUE TITLE
Feature request: accept png paths list as stdin to ease wrapper script creation

### ISSUE THREAD

**Issue by montehurd** [shssoichiro/oxipng#629]
If a _png paths list_ could be passed via stdin it would be easier to optimize a subset of a collection of images ( in my use case this subset is unoptimized images ) while still leveraging oxipng's great built-in thread spawning to churn through them efficiently

**Comment by ILoveGoulash** [comment]
That's what [xargs](https://pubs.opengroup.org/onlinepubs/9799919799/) is for.

**Comment by montehurd** [comment]
@ILoveGoulash Not trying to make N calls to it. Trying to let it handle thread spawning 

Unless I misunderstand what you are suggesting?

**Comment by ILoveGoulash** [comment]
Yes, `prints '%s\n' *.png | tr '\n' '\0' | xargs -0 oxipng` is (minus corner cases related to `ARG_MAX`) identical to `oxipng *.png`.

**Comment by murlakatamenka** [comment]
@montehurd just use [`fd`](https://github.com/sharkdp/fd):

```sh
# files will be processed 1 by 1 (i.e. not in parallel) and oxipng will be multithreaded
fd . --extension png --threads 1 --exec oxipng --preserve --opt max {}

# or with short flags
fd . -e png -j 1 -x oxipng -p -o max {}


# alternatively, run N threads for N files with single-threaded `oxipng`
fd . -e png -x oxipng -t 1 -p -o max {}
```
